packages/plugin-cache/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@xata.io/client-plugin-cache",
+  "version": "0.0.1",
+  "private": true,
```

Comment:
```suggestion
```

We can commit this change as soon as we agree on the naming for the SDK plugins

Reply:
What about `plugin-cache-memory`? The reason is:
- It'd be nice if all our plugins start with `plugin-`.
- And all the cache plugins start with `plugin-cache-` so it's easy to look for them alphabetically.
- I prefer "memory" because it makes it easy to understand that the caching is done in memory.

Reply:
Will we have ``plugins`` for the CLI? I'd like to differentiate between client and cli plugins if that's the case.

I kind of like the ``plugin-cache-*``, although maybe ``plugin-cache-lru`` instead of ``plugin-cache-memory``.

Reply:
Good point about the CLI having plugins. Then `client-plugin-cache` or `plugin-client-cache` makes sense to me.